tests/integration/semantics/src/test/java/org/apache/pulsar/tests/integration/functions/PulsarFunctionsTest.java

```diff
@@ -34,7 +34,6 @@
     // Tests on uploading/downloading function packages.
     //
 
-    @Test
     public String checkUpload() throws Exception {
```

Comment:
Why is this incorrect?

Reply:
test can only have a void method this test has never run
also download method already calls upload
```
    @Test
    public void checkDownload() throws Exception {
        String bkPkgPath = checkUpload();
        String localPkgFile = "/tmp/checkdownload-" + randomName(16);

        UploadDownloadCommandGenerator generator = UploadDownloadCommandGenerator.createDownloader(
                localPkgFile,
                bkPkgPath);

```

Reply:
Got it. Please leave these kind of comment in the PR description so that reviewer can understand the motivation and the mechanics of the change.